### PR TITLE
🐛 GREEN-46089: Add missing events endpoint to left nav menu

### PR DESCRIPTION
**Pull request by anirudhmungre**

### CODE REVIEW

File: source/includes/audit-log/_events.md [hunk 1]
```diff
@@ -16,7 +19,8 @@ event.type | One of the following values: `data_change_update`, `data_change_cre
 event.target_id | The ID of the element that was edited or accessed; this may be blank if the action does not target one particular ID
 event.target_type | The resource name for data changes, Harvest access, or the event action type for other actions
 event.meta | The before and after values from data change events, or other relevant data for the event such as the title of a report
-### GET: Retrieve events
+
+## GET: Retrieve events
```
Comment (anirudhmungre) [inline source/includes/audit-log/_events.md:23]: 3 # -> 1 # is the fix

File: .ruby-version
Comment (anirudhmungre) [inline .ruby-version:1]: This was annoying me, so I've added it in

File: source/includes/audit-log/_events.md (hunk 1 above)
Comment (julienguyenn) [reply source/includes/audit-log/_events.md:23]: Did you mean 3 # -> 2 # :P

**Review by julienguyenn — APPROVED** [review]

**Review by OmodaraAdonai — APPROVED** [review]
Looks good!

File: source/includes/audit-log/_events.md (hunk 1 above)
Comment (anirudhmungre) [reply source/includes/audit-log/_events.md:23]: lol yes 😅

**Pull request merged**